package.json

```diff
@@ -72,6 +72,7 @@
     "webpack4-bundle-tracker": "^0.0.1-beta",
     "winchan": "^0.2.1",
     "xdate": "^0.8.2",
+    "zulip-js": "^2.0.8",
```

Comment:
This should be in devDependencies, right?

Reply:
@aero31aero I did'nt find `zulip-js`anywhere in ` package.json`.

Reply:
@aero31aero If it has to be in `devDependencies`, I can amend the commit.

Reply:
Yeah, we shouldn't need the actual client library in production, so I agree with @aero31aero here.  We will want to be careful about any deps related to documentation generation, since that affects production, but I'm 99% sure we don't need any new deps for that.

Reply:
Apparently this commit was merged without being updated for this comment, so I moved zulip-js to devDependencies in #14969.